src/alarm.py

```diff
@@ -0,0 +1,29 @@
+import time
+from os import path, environ
+environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "hide"
+from pygame import mixer
```

Comment:
Can you set the environment value in `.env` file?

Reply:
Done